lib/tasks/vacancies_to_update.yaml

```diff
@@ -1226,3 +1226,42 @@ Telephone: 0191 2598500</p>
     -
       slug: 'teacher-with-phase-leader-responsibility-for-pupils-with-special-educational-needs-mainly-to-teach-in-ks3-ks4'
       ends_on: '22/10/2018'
+    -
+      slug: 'home-school-support-worker'
+      job_title: 'Home School Support Worker'
+      job_description: "<p>37 hours per week</p>
+      <p>Term time plus one week</p>
+      <p>Grade M point 25 – 29</p>
+      <p>The successful candidate will undertake casework working alongside school and locality staff to address the needs of the children, who need support to overcome barriers to learn both inside and outside school; this will including assessment and appropriate management of attendance.</p>
+      <p>Application forms and further details for the above position are available from our website www.educationvillage.org.uk Applicants are encouraged to visit The Education Village by contacting Helen Blanchard-Wynne on 01325 248173 to arrange an appointment.</p>
+      <p>Closing Date: 5.00pm on Friday 2nd November 2018</p>
+      <p>Interviews will be held w/c Monday 5th November 2018</p>
+      <p>The Education Village Academy Trust (EVAT) is committed to safeguarding the welfare of children and young people. You will be required to undertake an enhanced Disclosure check and Disqualification declaration.</p>"
+    -
+      slug: 'teacher-with-phase-leader-responsibility-for-pupils-with-special-educational-needs-mainly-to-teach-in-ks3-ks4'
```

Comment:
This needs to be merged with the previous entry for this Vacancy.

The reason for this is that the vacancy scraper will only update vacancies that have never been updated. This was put in place to ensure that we never override user changes, as a lot of schools who post to the service we scrape from also have access and use Teaching Jobs.